src/components/TabItem/TabItem.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tab: Tab;
+  selectedTabId: string;
+};
+
+export const TabItem: React.FC<Props> = ({
+  tab,
+  selectedTabId,
+}) => {
+  return (
+    <li
+      data-cy="Tab"
+      className={classNames(
+        { 'is-active': tab.id === selectedTabId },
+      )}
+      key={tab.id}
```

Comment:
Remove the `key` attribute from the `li` element since it's unnecessary in this component. It should be added to the mapped element while rendering a list outside this component. 😊

Reply:
Thank you, I forgot remove the `key` attribute from the `li` element when implemented the `TabItem` component.